web/sass/custom.scss

```diff
@@ -1,85 +1,120 @@
-$light-blue: #2d83ba;
-$orange: #F09740;
-$dark-blue: #053E4C;
-$grey: #242A2F;
-$text: #FFF;
-
-body {
-    color: $text;
-    font-family: "Lato", sans-serif;
-    font-size: 16px;
+//
+// Variables
+// --------------------------------------------------
+$light-blue: #2d83ba;  // $body-bg
+$dark-blue:  #052E4C;  // $brand-primary
+$orange:     #F09740;  // $brand-success
+$gray:       #242A2F;  // $gray
```

Comment:
We should be able to re-use the vars from bootstrap.  I'll take a look this weekend.